src/components/Uploader/style.scss

```diff
@@ -144,7 +143,7 @@
 }
 
 .proofOfAddressIcon {
-  &:extend(.identityIcon);
+  @extend .identityIcon;
```

Comment:
I don't think you're allowed to do that, linting does not complain? You should instead use classnames to add both classes on JSX.

Reply:
Have changed that in my latest commit. Didn't spot it initially as it was changed before your stylelint setup got merged in.